Technical-Workshop/Presentation/Home Screen/ViewModel/HomeScreenViewModelType.swift

```diff
@@ -10,4 +10,6 @@ import Foundation
 protocol HomeScreenViewModelType {
     var categoryItemsCount: Int { get }
     func categoryItem(at index: Int) -> CategoryItem
+    var mealsItemsCount: Int { get }
```

Comment:
let's arrange these according to type so it will be as following:
    var categoryItemsCount: Int { get }
    var mealsItemsCount: Int { get }
    func categoryItem(at index: Int) -> CategoryItem
    func mealItem(at index: Int) -> MealItem